sherpa/astro/tests/test_fake_pha.py

```diff
@@ -0,0 +1,227 @@
+#
```

Comment:
Ideally we could rely on the tests here and have a simple test at the UI layer to check it has passed the info through (since the UI layer no-longer does very much) but I don't have the energy to determine what UI tests could be removed now...

Reply:
Most of the UI tests are very short in run time (they fake 4 channels or so), so I decided to leave them in. At the very least, that guarantees backwards-compatibility in the sense that I did not assign an entirely new meaning to the setups already tested in the UI layer.